### DISCUSSION TITLE
Is there any way to list all relevant changed files in the "Folder History" view?

### DISCUSSION THREAD

**Discussion by rgeorgiev583** [gitkraken/vscode-gitlens#1978] [Q&A]
Right now, there is no way to view the changes in all files in a specific commit in the *File History* view after the *Open Folder History* command is used.  When a commit is chosen from the list in that view, only the changes in a single file from it are displayed.

I am considering opening an issue for this.

**Comment by eamodio** [comment]
Are you seeing something different in the File History view, when using the _Open Folder History_ command?

![image](https://user-images.githubusercontent.com/641685/165360785-6a91d4be-0794-435b-9768-a1e850e18381.png)

Each commit should be able to be expanded and each file can be diffed.

Was there something else you were expecting?

**Reply by rgeorgiev583** [reply]
> Are you seeing something different in the File History view, when using the _Open Folder History_ command?

Hmm, as far as I remember, the changes used to appear in that view even though I was using the _Open Folder History_ command...

> Each commit should be able to be expanded and each file can be diffed.

The commits are actually not expanding for me (there is no dropdown at all).  However, I guess that this could be because of something problematic in my specific setup.
I tried to use this feature in a remote devcontainer environment over SSH, which is quite a complex setup.  I'll try to reproduce the issue in a minimal, clean environment first.

> Was there something else you were expecting?

I expected things to happen just the way you described them.

**Reply by rgeorgiev583** [reply]
I just tested this feature again in my local environment (no devcontainer, no SSH), and it works just as you described it: after invoking the _Open Folder History_ command, the commits appear in the _Folder History_ view.  There is a dropdown for each commit, and it expands to show the list of changed files in the commit when clicked.
However, when I try to use the feature in my devcontainer setup over SSH, the commits are still shown in the _File History_ view, and they won't get expanded with a list of files when clicked: instead, only the diff of a single file (I couldn't determine which one) gets shown to the right.
I'm almost certain that the problem is somewhere in my environment (most probably, the VSCode server needs to be updated), so I don't think there is any point wasting your time with my issue anymore.  I'll try to recreate the setup from scratch and I'll check if there would be any difference.
Last but not least, thank you for the great work developing this extension and making working with Git much more comfortable for me :)

**Reply by eamodio** [reply]
❤️ Thank you! Please let me know what you find -- I would be really interested to know (and fix) any issues that you might find with the remote containers stuff. I would make sure that the GitLens version inside the container is up to date as well as VS Code. FYI, GitLens 12.0 requires VS Code version 1.64 or later.
